style_editor.html

```diff
@@ -10503,21 +10532,26 @@ <h4 class="settings-section-label" title="These settings are useful for people m
 }
 
 class SavedState {
-  constructor(name, def, update_function) {
+  constructor(name, def, update_function, type = "checkbox") {
```

Comment:
A more object-oreiented way to do this would be to have two subclasses: SavedStateBool and SavedStateNumber (or something like that). Both would inherit from SavedState and override just the parts required. (Which I guess would be the set() function.)

Reply:
ok. How about now?

Reply:
conflicts resolved...I think.